.env.example

```
@@ -0,0 +1,22 @@
+BETTER_AUTH_SECRET=your-better-auth-secret
+BETTER_AUTH_URL=http://localhost:3000 # Base URL of your app
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)